Octokit.Tests.Integration/fixtures/OrganizationsHooksCollection.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Xunit;
+
+namespace Octokit.Tests.Integration.fixtures
```

Comment:
I already changed the name on the other 2 classes as well should I undo the changes?